### ISSUE TITLE
"Got error while processing diagnostics: <command line>: cannot satisfy -package-id ..."

### ISSUE THREAD

**Issue by ceilican** [alanz/vscode-hie-server#91]
I have been trying to set up an IDE for Haskell in my Mac and Visual Studio Code with the Haskell Language Server extension is the one for which I came closest to success so far, but I get the following error:

```
Got error while processing diagnostics: <command line>: cannot satisfy -package-id Unique-0.4.7.2-JRWVZHedlBMCdzId2L2Yc1: Unique-0.4.7.2-JRWVZHedlBMCdzId2L2Yc1 is unusable due to missing dependencies: base-4.10.1.0 containers-0.5.10.2 extra-1.6.8-84f5CjUPNFBISJzUNl9auH hashable-1.2.7.0-LmYYwdRyxeo1gZdEWccp3y unordered-containers-0.2.9.0-AID53f8BSxj1wBLxJT9npu (use -v for more information)
```

All these dependencies are listed in my cabal file:

<img width="659" alt="screen shot 2018-06-12 at 08 19 12" src="https://user-images.githubusercontent.com/1809436/41271446-5d57b934-6e19-11e8-9b9f-b17da928d3a3.png">

How can I solve this problem?

**Comment by alanz** [comment]
This usually happens when the hie GHC version does not match the project GHC version.

Make sure you are using the wrapper to start the project.

**Comment by ceilican** [comment]
Yes.

I managed to solve the problem by using `stack install` with `stack-8.2.2.yaml` instead of the default `stack.yaml`.

As far as I know, GHC 8.2.2 is currently the latest stable version that is widely distributed. I think it would be more appropriate to let `stack.yaml` fetch from stable `lts` resolver, instead of the `nightly`. (It would have saved me a lot of time.)

Right now, following the instructions in this README (https://github.com/alanz/vscode-hie-server/blob/master/README.md) to install the requirements, I ended up with HIE compiled with GHC 8.4.3, and not with the GHC-8.2.2 that was required and used by everything else.

I would suggest to either change the resolver in `stack.yaml` of HIE (so that 8.2.2 is used by default) or improve the above mentioned README so that one knows what to do to compile HIE with the correct version of GHC.
